admin/templates/login.html

```diff
@@ -0,0 +1,70 @@
+{% load render_bundle from webpack_loader %}
+{% load staticfiles %}
+
+<!DOCTYPE html>
+<html>
```

Comment:
Is there a reason this doesn't extend base.html? You could save yourself a lot of html duplication.